cros_gralloc/cros_gralloc_driver.cc

```diff
@@ -109,6 +112,103 @@ cros_gralloc_driver *cros_gralloc_driver::get_instance()
         close(fd);                   \
     }
 
+int32_t cros_gralloc_driver::reload()
+{
+	int fd;
+	drmVersionPtr version;
+	char const *str = "%s/renderD%d";
+	const char *undesired[2] = { "vgem", nullptr };
+	uint32_t min_node = 128;
+	uint32_t max_node = (min_node + drv_num + 2);
```

Comment:
2 is magic number, it is better define one macro for it, for example:
#define IVSHMEME_DEVICE_NUM 2